src/views/ModelPlan/CollaborationArea/Cards/ModelPlanCard/index.test.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { GetModelPlanDocument, TaskStatus } from 'gql/gen/graphql';
+
+// import allMocks, {
+//   modelID,
+//   operationalNeedsMock,
+//   summaryMock
+// } from 'data/mock/readonly';
+import VerboseMockedProvider from 'utils/testing/MockedProvider';
+import setup from 'utils/testing/setup';
+
+import ModelPlanCard from './index';
+
+const modelID: string = 'f11eb129-2c80-4080-9440-439cbe1a286f';
+
+const modelPlanMocks = [
+  {
+    request: {
+      query: GetModelPlanDocument,
+      variables: { id: modelID }
+    },
+    result: {
+      data: {
+        modelPlan: {
+          __typename: 'ModelPlan',
+          id: modelID,
+          taskListStatus: TaskStatus.READY,
+          modifiedDts: '2021-08-10T14:00:00Z',
+          modifiedByUserAccount: {
+            __typename: 'UserAccount',
+            commonName: 'John Doe'
+          },
+          basics: {
+            __typename: 'PlanBasics',
+            status: TaskStatus.READY
+          },
+          generalCharacteristics: {
+            __typename: 'PlanGeneralCharacteristics',
+            status: TaskStatus.READY
+          },
+          participantsAndProviders: {
+            __typename: 'PlanParticipantsAndProviders',
+            status: TaskStatus.READY
+          },
+          beneficiaries: {
+            __typename: 'PlanBeneficiaries',
+            status: TaskStatus.READY
+          },
+          opsEvalAndLearning: {
+            __typename: 'PlanOpsEvalAndLearning',
+            status: TaskStatus.READY
+          },
+          payments: {
+            __typename: 'PlanBasics',
+            status: TaskStatus.READY
+          },
+          operationalNeeds: [
+            {
+              __typename: 'OperationalNeed',
+              modifiedDts: ''
+            }
+          ]
+        }
+      }
+    }
+  }
+];
```

Comment:
You test is generating a lot of warnings.  These should always attempt to be addressed.  You mock data should also be typed, which I think is generating all the errors in the console when running the test.  Once you type the mock data, TS will tell you what's needed from the mock